templates/user.html

```diff
@@ -15,21 +15,25 @@
   <meta property="og:description" content="See all your Open Source contributions in one place."/>
   <!-- Open Graph End -->
 
+  {{if (ne .ResponseType "iframe")}}
   <!-- Twitter Card Start -->
```

Comment:
@karanjthakkar when I made 2 templates you asked me to get rid of the second one to maintain only 1 template. Can we wrap up for now and decide importing and reactjs later? Fixed typo in README